kitsune/wiki/templates/wiki/review_revision.html

```diff
@@ -20,6 +20,15 @@
         <p>
           {{ _('This revision was already reviewed by {user} on {date}.')|fe(user=revision.reviewer, date=datetimeformat(revision.reviewed, format='longdatetime')) }}
         </p>
+      {% elif document.current_revision > revision %}
+        <p>
+          {{ _('A newer revision has already been reviewed.') }}
+          {% if latest_revision(revision, Q(is_approved=False)) > document.current_revision %}
+            <a href="{{ url('wiki.review_revision', document.slug, latest_revision(revision, Q(is_approved=False))) }}">{{ _('Please review the latest revision') }}</a>.
```

Comment:
`Q` is not defined in templates. You'll need to do this query in the view that calls this templates, and pass it in as context. This is why Travis failed when it ran the tests.